doc/user/content/sql/system-catalog.md

```diff
@@ -626,7 +626,7 @@ Field       | Type       | Meaning
 Materialize has compatibility shims for the following relations from [PostgreSQL's
 system catalog](https://www.postgresql.org/docs/current/catalogs.html):
 
-  * [`pg_access_methods`](https://www.postgresql.org/docs/current/catalog-pg-access-methods.html)
+  * `pg_access_methods`
```

Comment:
I think this is `pg_am`, judging from: https://github.com/MaterializeInc/materialize/blob/8e220981b933ec0bbec95a60e252498e5c23986b/src/coord/src/catalog/builtin.rs#L1961

Reply:
And there is a match in [Postgres pg_am](https://www.postgresql.org/docs/9.6/catalog-pg-am.html) for `relation access methods`